api/v1/db/json/notes-dao.js

```diff
@@ -7,22 +7,14 @@ const path = require('path');
 
 const { serializeNotes, serializeNote } = require('../../serializers/notes-serializer');
 
-const {
-  validateDBPath,
-  readJSONFile,
-  writeJSONFile,
-  initStudentDir,
-  getCounter,
-  incrementCounter,
-  deleteFile,
-} = appRoot.require('/utils/fs-operations');
+const fsOps = appRoot.require('utils/fs-operations');
+
+const { dbDirectoryPath } = config.get('api');
+fsOps.validateDBPath(dbDirectoryPath);
```

Comment:
This is good, but I'm thinking maybe we should validate the database path right after the API starts up not until we touch the `notes-dao`. What do you think?

Reply:
That sounds good to me. I'll change this.